src/ucp/rndv/rndv.c

```diff
@@ -1453,13 +1470,14 @@ ucs_status_t ucp_rndv_rts_handle_status_from_pending(ucp_request_t *sreq,
                (status != UCS_INPROGRESS));
 
     if (ucs_unlikely(status != UCS_OK)) {
+        ucp_request_send_untrack(sreq);
```

Comment:
we should not need to untrack when getting no_resource, only but when send really failed and we need to fail the request

Reply:
done